docs/release_notes/ibexa_dxp_v4.4.md

```diff
@@ -0,0 +1,120 @@
+
+#### Automated way of creating Personalization service account
+
+The Personalization service has been enhanced to speed up the process of creating new customer account.
+Now, what you have to do to create an account in the new, automated way, is to fill out the form, select an account type, and send request to the Personalization endpoint.
+Shortly after, you receive the credentials.
+
+For more information, see [documentation](https://doc.ibexa.co/projects/userguide/en/4.4/personalization/enable_personalization/#request-access-to-the-server).
+
+#### New models in Personalization engine
+
+Personalization engine introduces two new recommendation models: [predictive](https://doc.ibexa.co/projects/userguide/en/4.4/personalization/recommendation_models/#predictive) and [recurring purchase](https://doc.ibexa.co/projects/userguide/en/4.4/personalization/recommendation_models/#recurring-purchase). These two new models, based on mathematical approach, help to predict clients behavior and
+provide the best recommendations.
+
+## Ibexa Connect
+
+You can now take advantage of [Ibexa Connect](https://www.ibexa.co/products/ibexa-connect),
+an iPaaS (integration platform-as-a-service) which allows you to connect Ibexa DXP with different third-party applications.
+Ibexa Connect features a low-code drag-and-drop interface and hundreds of connectors to different services
+that help you automate business processes.
+
+See [Ibexa Connect documentation](https://doc.ibexa.co/projects/connect/en/latest/).
+
+![Example of an Ibexa Connect scenario](4.4_connect_scenario_example.png)
+
+## Other changes
+
+### Flysystem v2
+
+We have made significant upgrades to the codebase to rely on Flysystem v2.
+Our Flysystem Adapter implementation now supports dynamic paths
+described by complex settings resolvable for the SiteAccess context.
+For more information, see [Configuring the DFS IO handler](https://doc.ibexa.co/en/4.4/infrastructure_and_maintenance/clustering/clustering/#configuring-the-dfs-io-handler).
+
+If your custom project relies directly on a Flysystem features instead of using our IO abstraction,
+it will require an upgrade as well,
+using [these instructions](https://flysystem.thephpleague.com/docs/upgrade-from-1.x/).
+
+### Dedicated migration type for Corporate Accounts
+
+To simplify data migration, you can now create a corporate account with underling objects such as members group and address book.
+You can also extract those objects as references. 
+For more information on data migration actions, see [documentation](https://doc.ibexa.co/en/4.4/content_management/data_migration/data_migration_actions/#data-migration-actions).
+
+### API improvements
+
+### Item age in Recently added model
+
+Now, in a Recently added model (previously Random model), you can manually [set the age of items](https://doc.ibexa.co/projects/userguide/en/4.4/personalization/recommendation_models/#recently-added) which are displayed in recommendations.
+
+### Deprecations
```

Comment:
We also removed old/irrelevant v3 commerce blocks (e.g. bestsellers, product slider?) from Page builder.

Reply:
Very good point

Reply:
We were waiting for Adam's input with those packages as part of Commerce clean-up ticket. They are now added.